### DISCUSSION TITLE
Welcome to Victron2MQTT Discussions!

### DISCUSSION THREAD

**Discussion by softwarecrash** [softwarecrash/Victron2MQTT#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by buchi0815** [comment]
Hello, it is possible to connect three ve.direct. I have three Mppt 100/30. Unfortunately I can't program. It works very well with one.
bye

**Reply by all-solutions** [reply]
Unfortunately, this is not possible. VE.Direct is not a bus system. You will have to connect a Wemos to every Victron.

**Reply by buchi0815** [reply]
Hello, thank you for the quick reply. I know it's not a bus system. I thought to use the remaining GPIO.

bye

**Comment by hardyvo** [comment]
Hi, I think the TX/RX line (ESP to VE) is not used at all by your program. Is it possible to add a button to the website, that can be used to set the TX/RX to high level, so that the charger is switched OFF? Maybe a simpler way would be to enter a command in the webserial: chargerOFF or chargerON
The reason for this is because I have more than one charger and want to be able to switch any of them to OFF/ON.
By the way: it works really cool with a Victron Smart 100/20
